### DISCUSSION TITLE
附件过大导致崩溃

### DISCUSSION THREAD

**Discussion by cnjoby** [remotely-save/remotely-save#328] [General]
Remotely save 0.3.25
Obsidian： 1.4.3R 安卓版本
当选择超过50M的附件过滤，系统就正常，超过50M就会崩溃。

java.lang.outofmemoryerror: failed to allocate a 131065264 byte allocation with 15226992 free bytes and 14mb until oom, target footprint 268435456, growth limit 268435456 at java.lang.stringfactory.newstringfrombytes(stringfactory.java:86) at java.lang.stringfactory.newstringfrombytes(stringfactory.java:65) at android.util.base64.encodetostring(base64.java:458) at com.capacitorjs.plugins.app.appplugin.requesturlandroid(appplugin.java:249) at com.capacitorjs.plugins.app.appplugin.lambda$requesturl$2$com-capacitorjs-plugins-app-appplugin(appplugin.java:157) at com.capacitorjs.plugins.app.appplugin$$externalsyntheticlambda2.run(unknown source:4) at java.util.concurrent.threadpoolexecutor.runworker(threadpoolexecutor.java:1137) at java.util.concurrent.threadpoolexecutor$worker.run(threadpoolexecutor.java:637) at java.lang.thread.run(thread.java:1012)

**Comment by fyears** [comment]
没办法了，这里就是依赖系统内存做处理的，手机系统确实有点限制
